toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/rest/StopHandler.java

```diff
@@ -1,14 +1,16 @@
 package io.toast.tk.agent.web.rest;
 
-import org.vertx.java.core.Handler;
-import org.vertx.java.core.http.HttpServerRequest;
-
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.vertx.core.http.HttpServerRequest'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)